src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.h

```diff
@@ -261,7 +264,23 @@ class BtreeLBAManager : public LBAManager {
       actual_addr,
       intermediate_base,
       intermediate_length,
-      nullptr);
+      nullptr
+    ).si_then([&t, this, intermediate_base](auto indirect_mapping) {
+      assert(indirect_mapping->is_indirect());
+      return update_refcount(t, intermediate_base, 1, false
+      ).si_then([imapping=std::move(indirect_mapping)](auto p) {
+	auto mapping = std::move(p.second);
```

Comment:
Is it possible to assert that the direct-mapping/leaf-node's child, if present in cache/transaction, must be a stable extent?